build.sbt

```diff
@@ -35,7 +35,21 @@ lazy val commonSettings: Seq[sbt.Def.Setting[_]] = Seq(
 )
 
 lazy val commonJsSettings: Seq[sbt.Def.Setting[_]] = Seq(
-  Test / scalaJSLinkerConfig ~= { _.withModuleKind(ModuleKind.CommonJSModule) }
+  Test / scalaJSLinkerConfig ~= { _.withModuleKind(ModuleKind.CommonJSModule) },
+  scalacOptions ++= {
+    // Map the sourcemaps to github paths instead of local directories
+    val flag =
+      if (scalaVersion.value.startsWith("3")) "-scalajs-mapSourceURI"
+      else "-P:scalajs:mapSourceURI"
+    val localSourcesPath = baseDirectory.value.toURI
+    val headCommit       = git.gitHeadCommit.value.get
+    scmInfo.value.map { info =>
+      val remoteSourcesPath =
+        s"${info.browseUrl.toString
+            .replace("github.com", "raw.githubusercontent.com")}/$headCommit"
+      s"${flag}:$localSourcesPath->$remoteSourcesPath"
```

Comment:
Gosh. I would never have thought of that. Thanks!

Reply:
Same here, all credit to the excellent [sbt-typelevel](https://github.com/typelevel/sbt-typelevel) plugin